src/components/ReportActionItem/ReportPreview.tsx

```diff
@@ -140,14 +140,12 @@ function ReportPreview({
     const hasErrors = (hasReceipts && hasMissingSmartscanFields) || (canUseViolations && ReportUtils.hasViolations(iouReportID, transactionViolations));
     const lastThreeTransactionsWithReceipts = transactionsWithReceipts.slice(-3);
     const lastThreeReceipts = lastThreeTransactionsWithReceipts.map((transaction) => ReceiptUtils.getThumbnailAndImageURIs(transaction));
+    const hasOnlyPendingDistanceRequests = hasOnlyTransactionsWithPendingRoutes && !totalDisplaySpend;
```

Comment:
This is a confusing variable name. I suggest not using this variable at all and use `hasOnlyTransactionsWithPendingRoutes && !totalDisplaySpend;` in both of the places below.

Reply:
I honestly don't see what's wrong with `hasOnlyPendingDistanceRequests` as it's quite self-explanatory, but it's your call – I'll update 👌